ada_2_fhir/zibs2017/payload/zib-plannedcareactivityfortransfer.xsl

```diff
@@ -0,0 +1,432 @@
+        <xsl:param name="dateT" as="xs:date?"/>
+        
+        <xsl:param name="startDate" as="element()?"/>
+        <xsl:param name="endDate" as="element()?"/>
+        <xsl:param name="orderStatus" as="element()?"/>
+        
+        <xsl:for-each select="$in">
+            <xsl:variable name="resource">
+                <xsl:variable name="profileValue">http://nictiz.nl/fhir/StructureDefinition/eAfspraak-Appointment</xsl:variable>
+                <Appointment>
+                    <xsl:if test="string-length($logicalId) gt 0">
+                        <id value="{nf:make-fhir-logicalid(tokenize($profileValue, './')[last()], $logicalId)}"/>
+                    </xsl:if>
+                    <meta>
+                        <profile value="{$profileValue}"/>
+                    </meta>
+                    <xsl:for-each select="zibroot/identificatienummer | hcimroot/identification_number">
+                        <identifier>
+                            <xsl:call-template name="id-to-Identifier">
+                                <xsl:with-param name="in" select="."/>
+                            </xsl:call-template>
+                        </identifier>
+                    </xsl:for-each>
+                    
+                    <xsl:variable name="startDate" select="($startDate | begin_datum_tijd | start_date_time)[1]/@value"/>
+                    <xsl:variable name="endDate" select="($endDate | eind_datum_tijd | end_date_time)[1]/@value"/>
+                    
+                    <status>
+                        <xsl:attribute name="value">
+                            <!-- No ConceptMap known, see MM-4934 -->
+                            <xsl:choose>
+                                <xsl:when test="$orderStatus/@code = 'new'">proposed</xsl:when>
+                                <xsl:when test="$orderStatus/@code = 'active'">pending</xsl:when>
+                                <xsl:when test="$orderStatus/@code = 'held'">cancelled</xsl:when>
+                                <xsl:when test="$orderStatus/@code = 'cancelled'">cancelled</xsl:when>
+                                <xsl:when test="$orderStatus/@code = 'aborted'">noshow</xsl:when>
+                                <xsl:when test="$orderStatus/@code = '385644000'">proposed</xsl:when>
+                                <xsl:when test="$orderStatus/@code = '385651009'">arrived</xsl:when>
+                                <xsl:otherwise>
+                                    <xsl:attribute name="value" select="'unknown'"/>
+                                </xsl:otherwise>
+                            </xsl:choose>
+                        </xsl:attribute>
+                        <xsl:call-template name="ext-code-specification-1.0">
+                            <xsl:with-param name="in" select="$orderStatus"/>
+                        </xsl:call-template>
+                    </status>
+                    
+                    <xsl:if test="contact_type">
```

Comment:
Is there a reason this is no `for-each` statement as for the other elements? (If this is changed, make sure to change line 118 as well.)

Reply:
Probably copied from somewhere